resources/test/tests/idlharness/IdlInterface/test_to_json_operation.html

```diff
@@ -117,11 +113,11 @@
                 "status_string": "FAIL"
             },
             {
-                "message": "assert_unreached: property \"bar\" should not be present in the output of C.prototype.toJSON() Reached unreachable code",
+                "message": null,
                 "name": "Test default toJSON operation of C",
                 "properties": {},
-                "stack": "(implementation-defined)",
-                "status_string": "FAIL"
+                "stack": null,
+                "status_string": "PASS"
```

Comment:
Just checking, but are you also testing a fail condition? I've not checked the rest of the code in the test file, but I'm wondering as you are replacing a fail test with a pass test.

Reply:
This used to fail when extra properties were found in the outputs JSON. This creates issues with extension specs. Hence removing it. So we used to check for this failing condition, and now we want to make sure this passes instead of fails. Does that answer your question?

Reply:
Yep, thanks. I guess it’s not possible to check the union of the extension and the underlying definition for extraneous props? If no, that’s fine. Just wondering.

Reply:
Well, not automatically for now. Maybe once we have all WebIDL constructs in the interface folder will things change.